runTests.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+[ "$TEST_URL" == "" ] && echo "Error: cannot find TEST_URL env var." && exit 1
+
+[ "$TEST_HEATH_URL" == "" ] && TEST_HEALTH_URL="${TEST_URL}/health"
+
+# smoke or functional
+[ "$TASK" == "" ] && TASK="smoke"
+
+_healthy="false"
+
+for i in $(seq 0 30)
+do
+  sleep 10
+  wget $TEST_HEALTH_URL
+  [ "$?" == "0" ] && _healthy="true" && break
+done
+
+[ "$_healthy" != "true" ] && "Error: application does not seem to be running" && exit 2
```

Comment:
Seen a million times teams seeing an app not start up and sending jenkins builds urls, this probably won't stop it, but it may help
```suggestion
[ "$_healthy" != "true" ] && "Error: application does not seem to be running, check the application logs to see why" && exit 2
```